.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
The addition of `.nvmrc` made sense for nextstrain.org since we only support a single version of Node.js there, and `.nvmrc` serves to define that single version for devs who use nvm. However, Auspice supports a range of Node.js versions so there's no reason to constrain to a single version.